ssm_log_bucket.tf

```
@@ -1,5 +1,6 @@
 locals {
   account_id        = data.aws_caller_identity.current.account_id
+  aws_partition     = data.aws_partition.current.partition
```

After count logic is applied

```suggestion
  account_id        = join("", data.aws_caller_identity.current.*.account_id)
  aws_partition     = join("", data.aws_partition.current.*.partition)
```